src/state/index.ts

```diff
@@ -2,6 +2,9 @@ import { connectRouter, RouterState } from 'connected-react-router';
 import { createBrowserHistory, History } from 'history';
 import { combineReducers, createStore, Reducer } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
+import { persistReducer, persistStore } from 'redux-persist';
+import { default as autoMergeLevel1 } from 'redux-persist/es/stateReconciler/autoMergeLevel1';
+import { default as storage } from 'redux-persist/es/storage';
```

Comment:
Tbh this is the same as a default export - nothing we can do against this, just the lib having a different stance on default vs named exports

Reply:
Very unfortunate indeed. This does give us better code hinting/refactoring options. Ok if I keep it?

Reply:
Sure